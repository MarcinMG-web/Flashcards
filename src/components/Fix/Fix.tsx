import React from 'react';

const Fix: React.FC = (): JSX.Element => {
  return (
    <div className='loaded_page'>
      <h3 className='loaded_page-text'>
        {`We are adding new features and the app is currently unavailable. Please try again later! :)`}
        <div style={{ color: 'red', marginTop: '25px' }}>
          <i className='fa fa-cog fa-spin fa-3x fa-fw'></i>
        </div>
      </h3>
    </div>
  );
};

export default Fix;
