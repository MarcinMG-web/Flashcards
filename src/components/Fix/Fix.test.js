import { render } from '@testing-library/react';
import Fix from './Fix';

import { BrowserRouter } from 'react-router-dom';

const MockFix = () => {
  return (
    <BrowserRouter>
      <Fix />
    </BrowserRouter>
  );
};

describe('Fix Tests - render', () => {
  it('snapshot', () => {
    const tree = render(<MockFix />);

    expect(tree).toMatchSnapshot();
  });

  it('render without crashing', () => {
    render(<MockFix />);
  });
});
