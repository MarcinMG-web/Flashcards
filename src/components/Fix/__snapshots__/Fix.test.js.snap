// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fix Tests - render snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="loaded_page"
      >
        <h3
          class="loaded_page-text"
        >
          We are adding new features and the app is currently unavailable. Please try again later! :)
          <div
            style="color: red; margin-top: 25px;"
          >
            <i
              class="fa fa-cog fa-spin fa-3x fa-fw"
            />
          </div>
        </h3>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="loaded_page"
    >
      <h3
        class="loaded_page-text"
      >
        We are adding new features and the app is currently unavailable. Please try again later! :)
        <div
          style="color: red; margin-top: 25px;"
        >
          <i
            class="fa fa-cog fa-spin fa-3x fa-fw"
          />
        </div>
      </h3>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
